src/Microsoft.SqlTools.ServiceLayer/Admin/Contracts/GetBackupFolderRequest.cs

```diff
@@ -1,4 +1,4 @@
-﻿//
```

Comment:
Intentional?  Or should this be reverted?

Reply:
I'm not sure why this change is getting reverted? It seems the change was added somehow but cannot be reverted :(

Reply:
Life is mysterious sometimes 🙃